WebApp/autoreduce_webapp/selenium_tests/pages/help_page.py

```diff
@@ -0,0 +1,62 @@
+# ############################################################################### #
+# Autoreduction Repository : https://github.com/ISISScientificComputing/autoreduce
+#
+# Copyright &copy; 2021 ISIS Rutherford Appleton Laboratory UKRI
+# SPDX - License - Identifier: GPL-3.0-or-later
+# ############################################################################### #
+"""
+Module for the help summary page model
+"""
+from __future__ import annotations
+from typing import List
+
+from django.urls.base import reverse
+from selenium.webdriver.remote.webelement import WebElement
+
+from selenium_tests import configuration
+from selenium_tests.pages.component_mixins.footer_mixin import FooterMixin
+from selenium_tests.pages.component_mixins.navbar_mixin import NavbarMixin
+from selenium_tests.pages.page import Page
+
+
+class HelpPage(Page, NavbarMixin, FooterMixin):
+    """
+    Page model class for help page
+    """
+    def __init__(self, driver):
+        super().__init__(driver)
+
+    def url_path(self) -> str:
+        """
+        Return the current URL of the page.
+        :return: (str) the url path
+        """
+        return reverse("help")
+
+    def launch(self) -> HelpPage:
+        """
+        Open the page with the webdriver
+        :return: (HelpPage) The HelpPage object model
+        """
+        # Navigates to / first to force a login. Check the README and
+        # the "index" view for more details
+        self.driver.get(configuration.get_url())
+        self.driver.get(self.url())
+        return self
```

Comment:
This seems identical to the implementation already in [Page](https://github1s.com/ISISScientificComputing/autoreduce/blob/699-improve-help-page-search/WebApp/autoreduce_webapp/selenium_tests/pages/page.py#L19)

Reply:
I didn't notice that, thanks for spotting. In that case the launch methods here should be removed also: 
- `rerun_jobs_page.py`
- `run_summary_page.py`
- `runs_list_page.py`

Reply:
Yep I think we can remove them. If not the tests will fail 😄